### ISSUE TITLE
"concurrent connections per user exceeded" error when sending additional subscription messages

### ISSUE THREAD

**Issue by bobeasterday** [aisstream/issues#77]
According to the docs, changing the bbox is just a matter of sending another subscription request with the updated bbox. However, when I do so I get back an error ("concurrent connections per user exceeded") and the socket is closed. Should I not send the APIKey in the second request? Or maybe this is related to the recent load-shedding work...

**Comment by gbe668** [comment]
I observed the same thing. So i modified my code to close the socket, wait for 5 seconds and reopen the socket with the new bbox and the APIkey.

**Comment by bobeasterday** [comment]
Yeah. I was thinking I'd have to do the same in the absence of a fix.

**Comment by aisstream** [comment]
This appears to be a bug. We will look into this issue shortly.

**Comment by bobeasterday** [comment]
Awesome. Thanks for looking into it.

**Comment by bobeast** [comment]
As of this morning I'm now seeing "concurrent connections per user exceeded" even on the first subscribe message.
